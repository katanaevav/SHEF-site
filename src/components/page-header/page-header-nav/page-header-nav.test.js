import React from "react";
import renderer from "react-test-renderer";
import PageHeaderNav from "./page-header-nav.jsx";

it(`Render PageHeaderNav`, () => {
  const tree = renderer.create(
      <PageHeaderNav />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
