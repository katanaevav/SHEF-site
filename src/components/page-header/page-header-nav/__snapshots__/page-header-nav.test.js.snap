// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PageHeaderNav 1`] = `
<nav
  className="page-header__nav main-nav"
>
  <ul
    className="main-nav__menu main-menu"
  >
    <li
      className="main-menu__item"
    >
      <a
        href="#"
        title="Заказ еды"
      >
        Заказ еды
      </a>
      <ul
        className="main-menu__sub-menu sub-menu"
      >
        <li
          className="sub-menu__item"
        >
          <a
            href="catering.html"
            title="Кейтеринг"
          >
            Кейтеринг
          </a>
        </li>
        <li
          className="sub-menu__item"
        >
          <a
            href="online-cooking.html"
            title="Онлайн кулинария"
          >
            Онлайн кулинария
          </a>
        </li>
        <li
          className="sub-menu__item"
        >
          <a
            href="foods.html"
            title="Заказ продуктов"
          >
            Заказ продуктов
          </a>
        </li>
      </ul>
    </li>
    <li
      className="main-menu__item"
    >
      <a
        href="#"
        title="Бизнесу"
      >
        Бизнесу
      </a>
      <ul
        className="main-menu__sub-menu sub-menu"
      >
        <li
          className="sub-menu__item"
        >
          <a
            href="#"
            title="Доставка еды"
          >
            Доставка еды
          </a>
        </li>
        <li
          className="sub-menu__item"
        >
          <a
            href="#"
            title="Стань партнером"
          >
            Стань партнером
          </a>
        </li>
      </ul>
    </li>
    <li
      className="main-menu__item"
    >
      <a
        href="#"
        title="О нас"
      >
        О нас
      </a>
    </li>
  </ul>
</nav>
`;
